hazelcast/config.py

```diff
@@ -11,6 +11,10 @@
     none_type,
     try_to_get_enum_value,
 )
+try:
+    from .security import TokenProvider
```

Comment:
Why do we have such import?

Reply:
Importing `TokenProvider` is necessary for the type annotation. The `try` is a leftover. Fixed in: https://github.com/hazelcast/hazelcast-python-client/pull/446/commits/36946e76b40c9e937d31840499db6bd23f1b14c6